From: Simon McVittie <smcv@debian.org>
Date: Fri, 18 Nov 2022 11:21:18 +0000
Subject: build: Define USE_XDG_ACTIVATION conditional when Wayland is
 disabled

This fixes Autotools builds that disable Wayland, such as non-Linux
operating systems and the minimal "udeb" (micro-.deb) package used in
Debian's installer.

Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/5226
Applied-upstream: 3.24.35, commit:20f2c13b2b34397fbaf298c10537a842465ee7a3
---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index b17c5cc..ae4110e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -480,6 +480,7 @@ if test "$enable_wayland_backend" = "yes"; then
   AM_CONDITIONAL(USE_WAYLAND, true)
 else
   AM_CONDITIONAL(USE_WAYLAND, false)
+  AM_CONDITIONAL(USE_XDG_ACTIVATION, false)
 fi
 
 # strip leading space
